packages/fx-core/src/component/constants.ts

```diff
@@ -19,3 +19,29 @@ export const ComponentNames = {
   Function: "function",
   SimpleAuth: "simple-auth",
 };
+
+export const ActionNames = {
+  provision: "provision",
+  configure: "configure",
+  generateBicep: "generateBicep",
+};
+
+export const ActionTypeFunction = "function";
+export const ActionTypeCall = "call";
+export const ActionTypeGroup = "group";
+export const ActionTypeShell = "shell";
+
+export const AzureSqlOutputs = {
```

Comment:
maybe move to `azureSql/constants.ts`?

Reply:
The outputs will be used by other components. It is better to put them in common constants.